### DISCUSSION TITLE
How to look for backend side of code

### DISCUSSION THREAD

**Discussion by Vamshi3130** [Kanaries/Rath#423] [Q&A] [answered]
Hi , i want to use Rath backend services only for my project ,Could you please point me to where to look at ,also for AI part could i use my own llm ,if yes where to change that. Appreciate your response .Thank you in advance.

**Comment by ObservedObserver** [comment, marked as answer]
Backend services in open source version of RATH has not been maintained. Lots of automation features of EDA in EDA can work without a backend. I put a lot of effort to make sure RATH can handle most of the compuation at frontend with good performance.

The playground here [RATH online](https://rath.kanaries.net) actully runs all features at frontend.
